chain/rand/fake.go

```diff
@@ -0,0 +1,29 @@
+package rand
+
+import (
+	"context"
+	"math/rand"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/crypto"
+	"github.com/filecoin-project/lotus/chain/vm"
+)
+
+type fakeRand struct{}
+
+func NewFakeRand() vm.Rand {
+	return &fakeRand{}
+
+}
+
+func (fr *fakeRand) GetChainRandomness(ctx context.Context, personalization crypto.DomainSeparationTag, randEpoch abi.ChainEpoch, entropy []byte) ([]byte, error) {
+	out := make([]byte, 32)
+	_, _ = rand.New(rand.NewSource(int64(randEpoch * 1000))).Read(out) //nolint
```

Comment:
Why an error is ignored here? We could return it.

Reply:
Ideally, I wouldn't want these functions to fail, but I will add an error and remove in the future if it is annoying.